scripts/cleanup.sh

```diff
@@ -28,8 +28,11 @@ if [[ -n "$LOCAL" ]]; then
 
     sqlite3 test/config/ml-metadata/metadata.sqlite.db <<<"BEGIN TRANSACTION; $PARTIAL_SQL_CMD"
 else
-    echo 'Cleaning up kubernetes MySQL DB'
+    echo -n 'Cleaning up kubernetes MySQL DB...'
 
-    kubectl exec -n "$MR_NAMESPACE" -it "$(kubectl get pods -l component=db -o jsonpath="{.items[0].metadata.name}" -n "$MR_NAMESPACE")" \
-        -- mysql -u root -ptest -D "$TEST_DB_NAME" -e "START TRANSACTION; $PARTIAL_SQL_CMD"
+    kubectl exec -n "$MR_NAMESPACE" \
+        "$(kubectl get pods -l component=db -o jsonpath="{.items[0].metadata.name}" -n "$MR_NAMESPACE")" \
+        -- mysql -u root -ptest -D "$TEST_DB_NAME" -e "START TRANSACTION; $PARTIAL_SQL_CMD; COMMIT;"
```

Comment:
I split the command up into more lines (sorry its not clear what the exact change is here), but the only difference is the removal of the `-it` flag.